dotcom-rendering/src/web/components/FilterButton.importable.tsx

```diff
@@ -13,7 +13,7 @@ interface ButtonProps {
 	type?: TopicType;
 	count?: number;
 	format: ArticleFormat;
-	isActive: boolean;
+	isActive?: boolean;
```

Comment:
Does this need to be optional? Looking below I can't see why it wouldn't ever be true or fasle

Reply:
If `selectedTopics` is undefined then as things are currently, `isActive` would be undefined.

We could do this instead though which might be preferable:
```js
	const isActive = selectedTopics
						? selectedTopics.includes(buttonParams)
						: false;
						```

Reply:
Ok, updated to the above.